canonical-web-and-design/maas-ui
# fix(legacy): load tabs in the query params

## Done

- Fix the machine tabs to load the tab set in the url.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the machine list.
- Using the dropdown in the status columns click on 'See events' or 'See logs'.
- You should be taken to the machine details at the correct tab.

## Fixes

canonical-web-and-design/maas-ui#1597
